rio_tiler/utils.py

```diff
@@ -789,3 +789,21 @@ def cast_to_sequence(val: Optional[Any] = None) -> Sequence:
         val = (val,)
 
     return val
+
+
+def CRS_to_uri(crs: CRS) -> Optional[str]:
+    """Convert CRS to URI.
+
+    Code adapted from https://github.com/developmentseed/morecantile/blob/1829fe12408e4a1feee7493308f3f02257ef4caf/morecantile/models.py#L148-L161
```

Comment:
isn't morecantile a dependency of rio-tiler?

Reply:
sure? I am just saying you don't have to copy the code over since it's already a dependency so you can just import it. I don't think there is ever a situation where morecantile would not be installed with rio-tiler

Reply:
This version is also a bit different